DashAI/back/models/parameters/models_schemas/DecisionTreeClassifier.json

```diff
@@ -53,7 +53,7 @@
           "error_msg": "The 'max_features' parameter should be one of 'auto', 'sqrt', 'log2', 'null'.",
           "description": "The number of features to consider when looking for the best split",
           "type": "string",
-          "default": "l2",
+          "default": null,
           "enum": ["auto", "sqrt", "log2", null]
```

Comment:
Adding "null" in "enum" causes an empty option to exist in the dropdown of the parameter form.

If that parameter is configured, the default value cannot be re-selected.